packages/date-time-picker/src/DateTimePicker.ts

```diff
@@ -0,0 +1,874 @@
+import {
+    property,
+    query,
+    state,
+} from '@spectrum-web-components/base/src/decorators.js';
+import {
+    ClassInfo,
+    classMap,
+    ifDefined,
+    StyleInfo,
+    styleMap,
+    when,
+} from '@spectrum-web-components/base/src/directives.js';
+import { Calendar } from '@spectrum-web-components/calendar';
+import { ManageHelpText } from '@spectrum-web-components/help-text/src/manage-help-text.js';
+import {
+    LanguageResolutionController,
+    languageResolverUpdatedSymbol,
+} from '@spectrum-web-components/reactive-controllers/src/LanguageResolution.js';
+import { Focusable } from '@spectrum-web-components/shared/src/focusable.js';
+
+import styles from './date-time-picker.css.js';
+
+import '@spectrum-web-components/calendar/sp-calendar.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-alert.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-calendar.js';
+import '@spectrum-web-components/overlay/sp-overlay.js';
+import '@spectrum-web-components/picker-button/sp-picker-button.js';
+import '@spectrum-web-components/popover/sp-popover.js';
+
+import {
+    dateValueToDate,
+    equalSegmentValues,
+    isCalendarDate,
+    isCalendarDateTime,
+    isZonedDateTime,
+} from './helpers';
+import { DateTimeSegments } from './segments/DateTimeSegments';
+import { EditableSegment } from './segments/EditableSegment';
+import { LiteralSegment } from './segments/LiteralSegment';
+import { SegmentsFactory } from './segments/SegmentsFactory';
+import { ClearModifier } from './segments/modifiers/ClearModifier';
+import { DecrementModifier } from './segments/modifiers/DecrementModifier';
+import { IncrementModifier } from './segments/modifiers/IncrementModifier';
+import { InputModifier } from './segments/modifiers/InputModifier';
+import { type SegmentsModifierParams } from './segments/modifiers/SegmentsModifier';
+import {
+    DateTimePickerLabels,
+    EditableSegmentType,
+    Precision,
+    Precisions,
+    SegmentTypes,
+} from './types';
+
+/**
+ * @element sp-date-time-picker
+ *
+ * @event change - Announces when a new date/time is committed by the user
+ * @event input - Announces when the user's input changes the component's segments
+ *
+ * @slot calendar-icon - The icon used in the calendar button
+ * @slot help-text - Default or non-negative help text to associate to your form element
+ * @slot negative-help-text - Negative help text to associate to your form element when `invalid`
+ */
+export class DateTimePicker extends ManageHelpText(
+    SizedMixin(Focusable, {
+        noDefaultSize: true,
+    })
+) {
+    public static override get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    /**
+     * The selected date of the component. If defined, this also indicates where the calendar opens.
+     * If not, the calendar opens at the current month, and placeholder values are shown.
+     */
+    @property({ type: Object })
+    public value?: DateValue;
```

Comment:
Is it possible to set those properties `value` ,  `min`, `max` (also) as JS Date object to avoid having outside type dependencies to `@internationalized/date` ?